hoomd/hpmc/pytest/test_boxmc_move_tuner.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+from math import isclose
+import numpy as np
+import pytest
+
+from hoomd import hpmc
+from hoomd.conftest import operation_pickling_check
+from hoomd.hpmc.tune.boxmc_move_size import (_MoveSizeTuneDefinition,
+                                             BoxMCMoveSize)
+
+MOVE_TYPES = ("aspect", "volume", "shear_x", "shear_y", "shear_z", "length_x",
+              "length_y", "length_z")
+
+rng = np.random.default_rng(56456264)
```

Comment:
We should manage the numpy rng state globally with a pytest fixture in conftest.py.

Reply:
Fair point